src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/GetDescriptionPartsTest.kt

```diff
@@ -99,65 +99,77 @@ class GetDescriptionPartsTest {
 
       assertThat(result).isEqualTo(expected)
     }
-  }
 
-  @Test
-  fun `exception thrown when no valid date found in addendum`() {
-    val testDescription = "Original description" +
-      "User:STARK,TONY Date:Some updated details"
-    val minimalReportDto = createBasicReport(testDescription)
-
-    assertThatThrownBy { minimalReportDto.getDescriptionParts() }
-      .isInstanceOf(ValidationException::class.java)
-      .hasMessage(
-        "Validation issue: " +
-          "STARK,TONY Date:Some updated details",
-      )
+    @Test
+    fun `can accept null description`() {
+      val nomisReport = createBasicReport(null)
```

Comment:
technically, description can be null and it’s handled: so we ought to test for that too